source/d3d8types.hpp

```diff
@@ -5,6 +5,7 @@
 
 #pragma once
 
+struct IUnknown;
```

Comment:
Would prefer a `#include <unknwn.h>` after `#include <d3d9.h>` instead. Should work the same.